spark-cassandra-connector/src/main/scala/org/apache/spark/sql/cassandra/ConfCache.scala

```diff
@@ -0,0 +1,133 @@
+package org.apache.spark.sql.cassandra
+
+import com.datastax.spark.connector.cql.CassandraConnectorConf
+import com.datastax.spark.connector.rdd.ReadConf
+import com.datastax.spark.connector.writer.WriteConf
+import org.apache.commons.lang.StringUtils
+
+import scala.collection.mutable
+
+
+object ConfCache {
```

Comment:
First thing - why is this called `ConfCache`? Is this caching a configuration for something? What configuration? Why is it caching it? I wouldn't have to guess if:
1. the name was better
2. there was some ScalaDoc
(and the best would be to have both)